### PR TITLE
KiviAds Bid Adapter : initial release

### PR DESCRIPTION
**Pull request by KiviAds**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

**Comment by KiviAds** [comment]
Doc update PR - https://github.com/prebid/prebid.github.io/pull/4634

File: modules/kiviadsBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    const request = {
+      deviceWidth,
+      deviceHeight,
+      language,
+      secure,
+      host,
+      page,
+      placements,
+      coppa: config.getConfig('coppa') === true ? 1 : 0,
+      ccpa: bidderRequest.uspConsent || undefined,
```
Comment (patmmccann) [inline modules/kiviadsBidAdapter.js:155]: This is imminently or if date, why not get gpp?

File: modules/kiviadsBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+      gdpr: bidderRequest.gdprConsent || undefined,
+      tmax: config.getConfig('bidderTimeout')
```
Comment (patmmccann) [inline modules/kiviadsBidAdapter.js:157]: Please prefer the object

File: modules/kiviadsBidAdapter.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    const page = location.pathname;
+    const secure = location.protocol === 'https:' ? 1 : 0;
+    const placements = [];
+    const request = {
+      deviceWidth,
+      deviceHeight,
+      language,
+      secure,
+      host,
+      page,
```
Comment (patmmccann) [inline modules/kiviadsBidAdapter.js:152]: Why not just take the site.page value, I'm confused by how this is calculated

File: test/spec/modules/kiviadsBidAdapter_spec.js [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+        'ccpa',
+        'gdpr',
+        'tmax'
+      );
+      expect(data.deviceWidth).to.be.a('number');
+      expect(data.deviceHeight).to.be.a('number');
+      expect(data.language).to.be.a('string');
+      expect(data.secure).to.be.within(0, 1);
+      expect(data.host).to.be.a('string');
+      expect(data.page).to.be.a('string');
```
Comment (patmmccann) [inline test/spec/modules/kiviadsBidAdapter_spec.js:131]: Please write out exactly what string you expect

**Review by patmmccann — CHANGES_REQUESTED** [review]
Thanks!!

File: test/spec/modules/kiviadsBidAdapter_spec.js (hunk 4 above)
Comment (KiviAds) [reply test/spec/modules/kiviadsBidAdapter_spec.js:134]: did you mean like that?

File: modules/kiviadsBidAdapter.js (hunk 3 above)
Comment (KiviAds) [reply modules/kiviadsBidAdapter.js:152]: sorry, I didn't really understand what exactly needs to be changed, can you please give an example?

**Comment by KiviAds** [comment]
@patmmccann for some reason circleci build is failed, but i can`t open details. Is this ok?

File: modules/kiviadsBidAdapter.js (hunk 1 above)
Comment (patmmccann) [reply modules/kiviadsBidAdapter.js:155]: ccpa is about to be deprecated by the IAB. It was joing to be on july 1 but they pushed it to sept 30. You should add support for GPP if you expect publishers to use your adapter beyond sept 30

File: modules/kiviadsBidAdapter.js (hunk 3 above)
Comment (patmmccann) [reply modules/kiviadsBidAdapter.js:152]: i follow it now

**Review by patmmccann — APPROVED** [review]

**Pull request merged**